miio/gateway.py

```diff
@@ -646,71 +646,112 @@ class GatewayLight(GatewayDevice):
     """Light controls for the gateway."""
 
     @command()
-    def get_night_light_rgb(self):
-        """Unknown."""
-        # Returns 0 when light is off?"""
-        # looks like this is the same as get_rgb
-        # id': 65064, 'method': 'set_night_light_rgb', 'params': [419407616]}
-        # {'method': 'props', 'params':
-        # {'light': 'on', 'from.light': '4,,,'}, 'id': 88457} ?!
-        return self._gateway.send("get_night_light_rgb")
+    def rgb_status(self):
+        """Get current status of the light (always correct)."""
+        # Returns {"is_on": false, "brightness": 0, "rgb": (0, 0, 0)} when light is off
```

Comment:
I think this commentary could go directly to the docstring:
```
"""Get current status of the light.

Example:
   {"is_on": false, "brightness": 0, "rgb": (0, 0, 0)}
"""

It is also unclear what does "always correct" mean in this context, so that should be clarified (or removed, if unnecessary).

Reply:
done, you are right!